src/EmotiBitFactoryTest.cpp

```diff
@@ -1,5 +1,14 @@
 #include "EmotiBitFactoryTest.h"
 #ifdef ARDUINO
+#include <SPI.h>
+#include <Wire.h>
+#if defined ARDUINO_FEATHER_ESP32
+#include <SD.h>
+#else
+#include <SdFat.h>
+#endif
+#include "EmotiBit_Si7013.h"
+#include "SparkFun_External_EEPROM.h"
```

Comment:
Removed the includes.
It was because of a Library dependency finder setting in platform IO. Even though the files were in the include path, the compiler was not able to see it because of the dependency finder more. More information [here in the PlatformIO documentation](https://docs.platformio.org/en/latest/librarymanager/ldf.html#dependency-finder-mode).

The respective commit modifying the main project file can be found here:
1. https://github.com/EmotiBit/EmotiBit_FeatherWing/pull/244/commits/cb2ca07d4e63f1f617f97dc7a3b7f74b4d90c10e
2. https://github.com/EmotiBit/EmotiBit_FeatherWing/pull/244/commits/2919cad6c794a14c87d1bd6c39df945a7e98e624